Directory.Build.props

```diff
@@ -8,6 +8,7 @@
     <DebugType>embedded</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <ImplicitUsings>enable</ImplicitUsings>
+    <ContainerRepository>mcr.microsoft.com/dotnet/aspnet:8.0</ContainerRepository>
```

Comment:
This is going to make all of your apps create images called `mcr.microsoft.com/dotnet/aspnet:8.0`, what you probably meant is `<ContainerBaseImage>mcr.microsoft.com/dotnet/aspnet:8.0</ContainerBaseImage>`.